gym_roboy/visualize_agent.py

```diff
@@ -0,0 +1,46 @@
+import os
+import sys
+
+# Fail fast before importing stable-baselines
+import time
+
+MODEL_FILE = os.path.abspath(sys.argv[1])
+assert os.path.isfile(MODEL_FILE), "File not found: '{}'".format(MODEL_FILE)
+
+from stable_baselines.common.vec_env import DummyVecEnv
+from stable_baselines import PPO2
+
+from .envs import RoboyEnv, ROSBridgeProxy
+from .envs.robots import MsjRobot
+
+
+class Logger:
+    step = 0
+
+    def log(self, reward):
+        if self.step % 1 == 0:
+            print("reward:", reward)
+        self.step += 1
+        time.sleep(0.15)
+
+
+def main():
+    logger = Logger()
+    # The algorithms require a vectorized environment to run
+    env_constructor = lambda: RoboyEnv(ros_proxy=ROSBridgeProxy(robot=MsjRobot()))
+    env = DummyVecEnv([env_constructor])
+
+    agent = PPO2.load(MODEL_FILE)
```

Comment:
what if the model.pkl has been created through, for example, SAC?

Reply:
@ALPA-Industry-and-Technology Good question. Im not sure yet of the behavior when loading different saved models. Shall we make a new issue for that? Maybe its out of scope for this one.